src/main/scala/lms/transformation/mpi_nccl_transformation/distributedTensor2MPINCCLConv.scala

```diff
@@ -66,6 +64,18 @@ trait DistributeTensor2MPI_NCCLConv extends DistributeTensor2MPI_NCCLBase {
       }
   }
 
```

Comment:
similar to get_conv_descriptor. Issues: 
1) return type be CUDNN_CONV_DESCRIPTOR or TOP? I think former is better because unlike get_descriptor, the return type is known.
2) should rename this function and get_descriptor

Reply:
I think CUDNN_CONV_DESCRIPTOR is better, because some of the TypeLess API functions use `CUDNN_CONV_DESCRIPTOR` typed parameters.

Reply:
fixed in c8c4d89